modules/createHashHistory.js

```diff
@@ -33,7 +37,7 @@ var DefaultQueryKey = '_k';
 function createHashHistory(options={}) {
   var { queryKey } = options;
 
-  if (queryKey && typeof queryKey !== 'string')
+  if (typeof queryKey !== 'string')
```

Comment:
Just FYI, I've put this in [the docs](https://github.com/rackt/history/blob/c5c71abf612d322388b11d352496e412a6900c16/docs/README.md#caveats-of-hash-history).